servers/quarkus-server/pom.xml

```diff
@@ -165,6 +165,14 @@
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-container-image-jib</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-kubernetes</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-minikube</artifactId>
+    </dependency>
```

Comment:
can these go in a different module? similar to how we do the lambda? Also, why are they needed at all if the helm chart is already available?

Reply:
strictly speaking, this is independent from the Helm chart, but this is the fastest/cheapest way to generate a functioning kubernetes manifest for nessie without having to look through K8s APIs. I'm basically using the generated manifest and copy/paste it into the Helm chart stuff where some values are then templated.

Reply:
so in theory it could be behind a profile or something? It really only needs to be run when re-generating the helm templates?

Reply:
I put it behind a `k8s` profile

> It really only needs to be run when re-generating the helm templates?

it doesn't need to be run at all. everything can be done manually, but the generated k8s manifests are quite helpful as references